src/main/scala/firrtl/AddDescriptionNodes.scala

```diff
@@ -108,6 +108,7 @@ class AddDescriptionNodes extends Transform {
     val compMap = annos.collect {
       case DescriptionAnnotation(ComponentName(comp, ModuleName(mod, CircuitName(circ))), desc) =>
         (mod, comp, desc)
+      case DescriptionAnnotation(ReferenceTarget(_, m, _, r, _), desc) => (m, r, desc)
```

Comment:
It turns out the pass was being scheduled, it just wasn't finding any annotations it could consume! For whatever reason, everything that comes out of the earlier pass is a `ReferenceTarget`. Should they be collected here, or is there something I should be doing differently upstream?